crest/Assets/Crest/Crest/Scripts/Collision/VisualiseCollisionArea.cs

```diff
@@ -12,7 +12,7 @@ public class VisualiseCollisionArea : MonoBehaviour
 
         float[] _resultHeights = new float[s_steps * s_steps];
 
-        static float s_radius = 5f;
+        readonly float s_radius = 5f;
```

Comment:
Readonly values which are initialised with constant values can also remain static.

Reply:
Oops not sure why i did that. fixed now. thanks.